futures-channel/tests/mpsc.rs

```diff
@@ -401,7 +401,8 @@ fn stress_close_receiver_iter() {
             Some(r) => assert!(i == r),
             None => {
                 let unwritten = unwritten_rx.recv().expect("unwritten_rx");
-                assert_eq!(unwritten, i);
+                assert!(unwritten <= i + 1);
```

Comment:
This is necessary in order to prevent this test from flaking due to the race condition when writing into a closed channel.

Reply:
Skimming over it looks like the unbounded variant also has the same issue?

Reply:
Yes, they both have this issue.